eds4jinja2/adapters/base_data_source.py

```diff
@@ -75,6 +75,25 @@ def fetch_tree(self) -> Tuple[Optional[object], Optional[str]]:
             logger.exception(f"With {type(self).__name__}, failed tree fetching {str(self)}")
             return None, str(e)
 
+    def fetch_tree_raw(self) -> Tuple[Optional[object], Optional[str]]:
```

Comment:
Instead of creating a new function, I would add a parameter `return_string:bool=False` to the `fetch_tree` function. 
This parameter would control how the output is returned, as a python object or as a string.

Reply:
I thought about it as well as it is the quicker solution, but it is an anti-pattern to change the return type based on a parameter. If you wish I can add the parameter and delete this implementation.